pkg/systemd/services.html

```diff
@@ -62,19 +62,15 @@
       <div class="filter-group">
         <button id="create-timer" data-toggle="#timer-dialog" data-target="#timer-dialog" class="btn btn-primary" translate>Create Timer</button>
         <input type="search" id="services-text-filter" class="form-control" aria-label="Filter" placeholder="Filter by name or description..." translate="placeholder"/>
-        <div class="dropdown" id="services-dropdown">
-          <button class="btn btn-default dropdown-toggle" type="button" id="service-type" data-toggle="dropdown">
-            <span id="current-service-type" class="pull-left" data-num=0 translatable="yes">All</span>
-            <span class="caret pull-right"></span>
-          </button>
-          <ul class="dropdown-menu" role="menu" aria-labelledby="service-type">
-            <li role="presentation"><a role="menuitem" data-num=0 translatable="yes">All</a></li>
-            <li role="presentation" class="divider"></li>
-            <li role="presentation"><a role="menuitem" data-num=1 translatable="yes">Enabled</a></li>
-            <li role="presentation"><a role="menuitem" data-num=2 translatable="yes">Disabled</a></li>
-            <li role="presentation"><a role="menuitem" data-num=3 translatable="yes">Static</a></li>
-          </ul>
-        </div>
+
+        <label id="services-dropdown-label" translatable="yes" class="hidden">Status</label>
```

Comment:
`Status` is not a great label. It should be most likely `Automatic start` or `Automatic start status` or something similar, but lets have there `automatic start` as it really reefers to it (also see the name of the column)

Reply:
s/start/startup

Reply:
Oh, hey! Here's an idea: We could even use the same string as the label in the `th` of the table for the `aria-labelledby` and drop the hidden `<label>`, as the select is conceptually tied to that column in the list!

Thankfully, `aria-labelledby` still works if the element is hidden (as long as it still exists n the DOM):
https://developer.paciellogroup.com/blog/2015/05/short-note-on-aria-labelledby-and-aria-describedby/

Reply:
Sounds good :)

Reply:
Oh, we didn't actually implement this change yet. Oops! :grin: